traefik/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   # -- Traefik image repository
   repository: traefik
   # -- defaults to appVersion
-  tag: ""
+  tag:
```

Comment:
Shouldn't it be better to use `~` instead of empty? Also, please update `VALUES.md`

Reply:
Thanks for your review. I updated `VALUES.md`.
I'm not sure I understand what you mean by using `~`. Would you please detail what you have in mind ?

Reply:
In the YAML, there are many ways to pass the `null` value; one is using the `~` character.

Reply:
\o thanks, I was not aware of it.

We are nullifying many values with empty. 
It's more readable, more common in other charts and is ready to be customized when user needs to.

=> So I think we are gonna stick with empty for this PR. 

Feel free to open a proposal if you think this is important.